massgov/openmass
# DP-23218: Add collections feature to  other content types

**Description:**
Added Collections field to Advisory, Binder, Curated List, Decision, Executive Order, Form, Guide, Location, Service, Service Details, Regulation, and Rules of Court. Configured the Listing display and added/updated template files.

**Jira:** (Skip unless you are MA staff)
DP-23218


**To Test:**
- Login and go to /admin/content?title=_qa
- Edit a node for each new type: Advisory, Binder, Curated List, Decision, Executive Order, Form, Guide, Location, Regulation, Rules of Court, Service, and Service Details.
- If there is no short description or listing description, add one.
- Select topics or sub topics in the "Collections" field for the "QAG Test Collection" collection.
- Save the nodes.
- Go to /collections/qagtest.
- Verify each content type example displays as expected.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
